plugins/gatsby-source-jenkinsplugins/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
this file needed?

Reply:
Nope, left behind files from the source I forked it from